README.adoc

```diff
@@ -31,12 +43,15 @@ npm run ionic:ios // to run in an ios
 ```
 
 ==== Troubleshooting
+
+===== Working with iOS and Android together
 When running the application on both android or ios you will need to clean the projects root directory to remove conflicting files specific to each platform
 This can be done by running the following:
 ```
 npm run clean
```

Comment:
I think we should instruction on how to get the sources before explaining how to compile.
Something like:
"To get the sources run `git clone xxx`"

Reply:
done